app/src/main/res/layout-land/profile_chooser_add_view.xml

```diff
@@ -7,8 +7,8 @@
     <import type="android.view.View" />
 
     <variable
-      name="presenter"
-      type="org.oppia.app.profile.ProfileChooserFragmentPresenter" />
+      name="wasProfileEverBeenAddedValue"
```

Comment:
Nit: ``wasProfileEverAdded`` or ``hasProfileEverBeenAdded``. Ditto elsewhere in PR.

Reply:
DOne!